src/Cogworks.FindAndReplace/Web/Components/FindAndReplaceComponent.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Umbraco.Core.Composing;
+using Umbraco.Web.Trees;
+
+namespace Cogworks.FindAndReplace.Web.Components
+{
+    public class FindAndReplaceComponent : IComponent
+    {
+        public void Initialize()
+        {
+            TreeControllerBase.MenuRendering += FindAndReplace_MenuRendering;
+        }
+
+        private static void FindAndReplace_MenuRendering(TreeControllerBase sender, MenuRenderingEventArgs e)
+        {
+            if (sender.TreeAlias != "content") return;
+
+            var menuItem = new Umbraco.Web.Models.Trees.MenuItem("findAndReplace", "Find and Replace")
+            {
+                AdditionalData =
+                {
+                    { "actionView", "/App_Plugins/FindAndReplace/Views/findandreplace.html" },
```

Comment:
Might be worth moving those strings (names, locations etc.) into Constants for the package lib, but it's minor and no showstopping now.